src/wp-includes/class.wp-dependencies.php

```diff
@@ -370,6 +370,7 @@ protected function recurse_deps( $queue, $handle ) {
 		$all_deps = array_fill_keys( $queue, true );
 		$queues   = array();
 		$done     = array();
+		$old_deps = $all_deps;
```

Comment:
Left over/typo?

Reply:
soudns like it, or a rebase gone wrong – good catch!